platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx

```diff
@@ -51,8 +53,11 @@ const ThumbnailNoImage = ({
             </Tooltip>
             <span className="ml-4 text-base text-blue-300">{seriesDate}</span>
           </div>
-          <div className="ml-12 text-base text-white break-all">
-            {description}
+          <div className="flex flex-row">
+            {canReject && <Icon name="old-trash" className="ml-4 w-3 text-red-500" onClick={onReject} />}
+            <div className="ml-4 text-base text-white break-all">
+              {description}
+            </div>
```

Comment:
The UI is not visible so I can't test, but we would want to make sure that the "delete" copy + icon are not a part of the drag/ghost image for the thumbnail card.

If it currently is, fine to kick this to a separate backlogged issue (enhancement, future)